pkg/cmd/cli/nodeagent/server.go

```diff
@@ -358,6 +357,16 @@ func (s *nodeAgentServer) run() {
 		if err := dataDownloadReconciler.AttemptDataDownloadResume(s.ctx, s.logger.WithField("node", s.nodeName), s.namespace); err != nil {
 			s.logger.WithError(errors.WithStack(err)).Error("Failed to attempt data download resume")
 		}
+
+		if err := pvbReconciler.AttemptPVBResume(s.ctx, s.logger.WithField("node", s.nodeName), s.namespace); err != nil {
```

Comment:
The cache waiting for PVB and PBR should be added in the function `waitCacheForResume`.